README.md

```diff
@@ -1,23 +1,78 @@
 # generator-polymer-init-custom-build
-> A starting point for building apps using polymer-build
 
-## Installation
+This template is a starting point for building apps using Polymer Starter Kit
+with a custom gulp process leveraging polymer-build, the library powering Polymer CLI
```

Comment:
Also add info about the additional functionality the gulpfile provides that the Polymer CLI does not, such as imagemin, clean, and clean before build.